content/working-groups/prioritization/FAQ.md

```diff
@@ -0,0 +1,9 @@
+---
+title: FAQs
+type: docs
+---
+# Frequently Asked Questions (FAQ)
+
+If you have a question, feel free to file an issue or ask in the working group's Zulip stream.
+
+There are no FAQs so far. Please ask us more questions.
```

Comment:
we probably don't need this file here -- I think the template should be slimmed down

Reply:
Ok going to remove this file.